addons/addon-base-raas-ui/packages/base-raas-ui/src/parts/accounts/AccountCard.js

```diff
@@ -199,20 +210,16 @@ class AccountCard extends React.Component {
   }
 
   renderUpdatePermsButton() {
-    const needsUpdate = this.needsUpdate;
+    const permissionStatus = this.permissionStatus;
     const buttonArgs =
-      needsUpdate === true
-        ? { message: 'Update Permissions', color: 'orange' }
-        : { message: 'Onboard Account', color: 'purple' };
-    // This button is only displayed if needsUpdate is either True or undefined
+      permissionStatus === 'NEEDSUPDATE'
```

Comment:
I would suggest using if/else or switch if there's two layers of ? : for better readability

Reply:
will do!